src/functionaltests/java/com/ericsson/ei/rules/RulesHandlerSteps.java

```diff
@@ -86,8 +90,9 @@ public void rules_are_loaded_with_exception() throws Exception {
      */
     private void initializeRulesHandler() throws Exception {
         LOGGER.debug("Rules Path: " + rulesPath);
-        System.setProperty("rules.path", rulesPath);
+//        System.setProperty("rules.path", rulesPath);
```

Comment:
If the commented line is not needed please remove it.

Reply:
done